src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurSaeCreateRequestedCashAdvanceFileValidationServiceImpl.java

```diff
@@ -0,0 +1,398 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ListIterator;
+
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
we should use the org.apache.commons.collections4.CollectionUtils version

Reply:
Changed.